spec/models/org_participation_spec.rb

```diff
@@ -3,4 +3,32 @@
 RSpec.describe OrgParticipation, type: :model do
   it { should belong_to(:activity) }
   it { should belong_to(:organisation) }
+
+  context "when role is implementing and organisation is inactive" do
+    subject(:org_participation) { build(:org_participation, :inactive_organisation) }
+
+    it "should not be valid" do
+      expect(org_participation.valid?).to be false
```

Comment:
Is this going to invalidate any existing records when an organisation becomes inactive? We should prevent invalidating any existing records.